Virtual Signals, Dynamic Locales, Better Icon retrieval

Nexela opened the pull request:
Still working on it but feel free to check it out.

So far I am at 80%+ dynamic locales and icons on about 150 mods.

Nexela committed:
Virtual Signals, Dynamic Locales, Better Icon retrieval

theRustyKnife commented:
Looks good ;)
I didn't know you can specify localised_name in the prototype definitions - Makes me feel kinda stupid now that I spent so much time doing the locale the way I did :/
I hope to get some work done on this soon™ so I'll check it out in more detail then.

Nexela committed:
Add seperate category

Nexela committed:
Colored the combinators to make them distinct
Add a new category for the virtual recipes
Prepend the virtual recipe locales with "recipe"

Nexela commented:
More goodies added.  Still looking into the few recipes with missing icons and a couple showing up as "something went wrong"

Nexela committed:
Even better icon checking
Fix typo in icon checking for some icons tables

Nexela committed:
Fix for Angels empty recipes
Most locales show up correctly

Nexela committed:
Fix makefile
